content/zh/docs/getting-started/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: 开始使用
+description: 根据你的角色开始使用 OpenTelemetry。
+no_list: true
+weight: 160
+default_lang_commit: d54bd689
```

Comment:
> HASH	ERROR	content/zh/docs/getting-started/_index.md: git diff error (128) or invalid hash d54bd689

This is not a valid hash of `main`, please update it -- see, https://github.com/open-telemetry/opentelemetry.io/commits/main/ for valid hashes.